### PR TITLE
Add bitwise

### PR DESCRIPTION
**Pull request by mellowcandle**
#### New App Submission

**App name:**
bitwise
**Repo or homepage link:**
https://github.com/mellowcandle/bitwise
**Description:**
Terminal based bitwise calculator in curses.
**Why you think it's awesome:**
First, I created it so I'm not objective, but it's very useful to me and my colleagues.
we manipulate bitmasks a lot, and switch between hex/dec representation.
We prefer to stay on terminal and not use web or desktop calculator.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -119,6 +119,7 @@ Inspired by the [awesome](https://github.com/sindresorhus/awesome) list thing.
 ## Development
 
 - [bcal](https://github.com/jarun/bcal) - Byte CALculator for storage conversions and calculations.
+- [bitwise](https://github.com/mellowcandle/bitwise) - Terminal based bit manipulator in ncurses.
```
Comment (jneidel) [inline readme.md]: As stated in the [contribution guidelines](https://github.com/agarrharr/awesome-cli-apps/blob/master/contributing.md#contribution-guidelines):
**Additions should be added to the bottom of the relevant category.**

**Comment by jneidel** [comment]
@mellowcandle Thanks for your submission!
If you fix the requested changes I'll get this PR merged :smile:

File: readme.md (hunk 1 above)
Comment (jneidel) [reply readme.md]: Could you also strip down the description to remove unnecessary information:
E.g.: `Terminal based bit manipulator in ncurses.` ->
`Bit manipulator in ncurses.`
This list is about clis, being terminal based is a given :)

**Comment by mellowcandle** [comment]
Done.
Thanks.

**Pull request merged**